app/mainwindow.cpp

```diff
@@ -480,14 +479,12 @@ void MainWindow::lastUsedTheme()
     currentTheme = themeCollection->theme(themeName);
     applyCurrentTheme();
 
-    for (auto action : stylesGroup->actions()) {
-        if (action->text() == themeName) {
-            action->setChecked(true);
-            break;
-        }
-    }
-
-    styleLabel->setText(themeName);
+	foreach(QAction* a, stylesGroup->actions()) {
```

Comment:
Previous version was always giving me segmentation faults, I see that you fixed it, but it was overwritten by my fix on merge, I preffer to use your version.